src/main/java/roomescape/auth/repository/JdbcMemberRepository.java

```diff
@@ -0,0 +1,99 @@
+package roomescape.auth.repository;
+
+import org.springframework.dao.EmptyResultDataAccessException;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.jdbc.core.RowMapper;
+import org.springframework.jdbc.core.namedparam.MapSqlParameterSource;
+import org.springframework.jdbc.core.namedparam.NamedParameterJdbcTemplate;
+import org.springframework.jdbc.support.GeneratedKeyHolder;
+import org.springframework.stereotype.Repository;
+import roomescape.auth.entity.Member;
+
+import java.util.List;
+import java.util.Optional;
+
+@Repository
+public class JdbcMemberRepository implements MemberRepository {
+    private static final RowMapper<Member> rowMapper = (resultSet, rowNum) ->
+            new Member(
+                    resultSet.getLong("id"),
+                    resultSet.getString("name"),
+                    resultSet.getString("role"),
+                    resultSet.getString("email"),
+                    resultSet.getString("password"));
+    private final NamedParameterJdbcTemplate jdbcTemplate;
```

Comment:
사소할 수 있지만 상수와 상태는 줄바꿈을 통해서 구분하기 쉽게해주면 코드 파악이 조금 더 쉬울 것 같아요. 😃 

```suggestion
                    resultSet.getString("password"));
                    
    private final NamedParameterJdbcTemplate jdbcTemplate;
```